codec/codec_test.go

```diff
@@ -2528,6 +2528,172 @@ func TestJsonInvalidUnicode(t *testing.T) {
 	}
 }
 
+type UBase struct {
+	S string
+	B bool
+	I int
+}
+
+type U1 struct {
+	UBase
+
+	UnknownFieldSetHandler
+}
+
+var _ UnknownFieldHandler = (*U1)(nil)
+
+type U2 struct {
+	UBase
+
+	S2 string
+	B2 bool
+	I2 int
+
+	UnknownFieldSetHandler
+}
+
+var _ UnknownFieldHandler = (*U2)(nil)
+
+func doTestEncUnknownFields(t *testing.T, h Handle) {
+	testOnce.Do(testInitAll)
```

Comment:
This file is unchanged, except for adding this line (which I opened up https://github.com/keybase/go-codec/pull/9 for separately)